cmd/cosign/cli/copy/copy.go

```diff
@@ -39,6 +40,10 @@ func CopyCmd(ctx context.Context, regOpts options.RegistryOptions, srcImg, dstIm
 	if err != nil {
 		return err
 	}
+	if _, ok := srcRef.(name.Digest); !ok {
+		msg := fmt.Sprintf(ui.TagReferenceMessage, srcImg)
+		ui.Warnf(ctx, msg)
+	}
```

Comment:
What would be the better recommendation to copy multi-arch images with signature if not using tag?

Reply:
Good point. I think that `copy` and `save` may have been collateral damage here.

Reply:
https://github.com/sigstore/cosign/pull/2799